rsyslog/init.sls

```diff
@@ -1,9 +1,10 @@
 {%- if pillar.rsyslog is defined %}
 include:
 {%- if pillar.rsyslog.server is defined %}
```

Comment:
Needs update:
- "rsyslog" is always defined I guess
- "server" is always defined (at leas as empty {} in map.jinja)
- do not use pillar.rsyslog, you may use `server` as you import on a first line

@fpytloun pls decide/what rules we have for `is defined` vs.`server.enabled`